testbed/testbed/scripts/ui.py

```diff
@@ -4,6 +4,7 @@
 class Layout:
     def __init__(self):
         self.testcase = widgets.Text(description="Testcase")
```

Comment:
We could maybe even remove this flag and add `tcp-transfer` as a protocol instead of a full-fledged test-case. Let's keep it in mind for the next refactor :)